### DISCUSSION TITLE
DataTable and Segmented can not be installed

### DISCUSSION THREAD

**Discussion by ghost1372** [CommunityToolkit/Labs-Windows#488] [Q&A] [answered]
Hi @michael-hawker ,  i can not install DataTable and SegmentedControl from Labs:

Installed CommunityToolkit.Labs.WinUI.Controls.DataTable 0.1.230809 from https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json with content hash QdoT686WxgttBZSPC5VCuQi1OsJ/qd/VyD0MrVVSZ+VUxyWlB0W/i7bi31/rKdxfsgn6TiodyECGQkYbUliVWQ==.

NU1101: Unable to find package CommunityToolkit.WinUI.Controls.Sizers. No packages exist with this id in source(s): Devexpress, github, Labs, Microsoft Visual Studio Offline Packages, nuget.org, WinUICommunity Local

NU1101: Unable to find package CommunityToolkit.WinUI.Converters. No packages exist with this id in source(s): Devexpress, github, Labs, Microsoft Visual Studio Offline Packages, nuget.org, WinUICommunity Local

NU1101: Unable to find package CommunityToolkit.WinUI.Extensions. No packages exist with this id in source(s): Devexpress, github, Labs, Microsoft Visual Studio Offline Packages, nuget.org, WinUICommunity Local

Package restore failed. Rolling back package changes for 'AlAnvar'.

**Comment by insomniachi** [comment, marked as answer]
They have dependency on the new Windows Community Toolkit preview packages 
https://github.com/CommunityToolkit/Windows

**Reply by ghost1372** [reply]
is there any packages for Windows? where?

**Reply by insomniachi** [reply]
https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json

**Reply by michael-hawker** [reply]
Thanks @insomniachi for jumping in here, yes this is the case at the moment as we move towards releasing our 8.0 packages. We were hoping for a bit quicker turn around but have been running into some issues as everything comes together.

Once we ship 8.0 packages to NuGet, we'll setup all the Labs packages to use those from the NuGet sources directly which will make them not require both feeds again.
